feature/interface/loopback/ate_tests/loopback_aggregate_test/README.md

```diff
@@ -0,0 +1,65 @@
+# RT-5.6: Interface Loopback mode
+
+## Summary
+
+Ensure Interface mode can be set to loopback mode and can be added as part of static LAG.
+
+## Procedure
+
+### TestCase-1:
+
+*   Configure DUT port-1
+*   Configure “loopback mode” set to “FACILITY”
+*   Add port-1 as part of Static LAG (lacp mode static(on))
+*   Validate that port-1 operational status is “UP”
+*   Validate that  LAG port status is “UP”
+
+### TestCase-2:
+
+*   Configure DUT port-1
+*   Configure “loopback mode” set to ”TERMINAL”
+*   Add port-1 as part of Static LAG (lacp mode static(on))
+*   Validate that port-1 operational status is “UP”
```

Comment:
As per OpaneConfig: 
"
TERMINAL
A loopback which directs traffic received from an external source on the port back out the transmit side of the same port.
"
This loopback mode (in contrast fo FACILITY) do not define status on local port, hence I would remove this TC at all in order to avoid ambiguity.

Reply:
for completeness of test, we need this test. moved to TBD as discussed offline with Rafal.

Reply:
also, removed TestCase-3 as its covered already in singleton tests